charts/jenkins/VALUES.md

```diff
@@ -38,32 +38,33 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | [agent.livenessProbe](./values.yaml#L941) | object |  | `{}` |
 | [agent.maxRequestsPerHostStr](./values.yaml#L896) | string | The maximum concurrent connections to Kubernetes API | `"32"` |
 | [agent.namespace](./values.yaml#L902) | string | Namespace in which the Kubernetes agents should be launched | `nil` |
-| [agent.nodeSelector](./values.yaml#L1038) | object | Node labels for pod assignment | `{}` |
+| [agent.nodeSelector](./values.yaml#L1043) | object | Node labels for pod assignment | `{}` |
 | [agent.nodeUsageMode](./values.yaml#L914) | string |  | `"NORMAL"` |
 | [agent.podLabels](./values.yaml#L904) | object | Custom Pod labels (an object with `label-key: label-value` pairs) | `{}` |
-| [agent.podName](./values.yaml#L1056) | string | Agent Pod base name | `"default"` |
-| [agent.podRetention](./values.yaml#L954) | string |  | `"Never"` |
-| [agent.podTemplates](./values.yaml#L1113) | object | Configures extra pod templates for the default kubernetes cloud | `{}` |
+| [agent.podName](./values.yaml#L1061) | string | Agent Pod base name | `"default"` |
+| [agent.podRetention](./values.yaml#L959) | string |  | `"Never"` |
+| [agent.podTemplates](./values.yaml#L1118) | object | Configures extra pod templates for the default kubernetes cloud | `{}` |
 | [agent.privileged](./values.yaml#L924) | bool | Agent privileged container | `false` |
 | [agent.resources](./values.yaml#L932) | object | Resources allocation (Requests and Limits) | `{"limits":{"cpu":"512m","memory":"512Mi"},"requests":{"cpu":"512m","memory":"512Mi"}}` |
+| [agent.restrictedPssSecurityContext](./values.yaml#L956) | bool |  | `false` |
```

Comment:
This is missing a description, can you add what's needed to make it show up here please (I think a one liner with -- in values.yaml)